Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx

```diff
@@ -71,6 +75,39 @@ const CreateNewMaintenanceWindow = () => {
     }));
   };
 
+  const handleSubmit = async () => {
+    const data = {
+      repeat: values.repeat,
+      date: values.date.format("YYYY-MM-DD"),
+      startTime: values.startTime.format("HH:mm"),
+      duration: values.duration,
+      unit: values.unit,
+      friendlyName: values.friendlyName,
+      addMonitors: values.AddMonitors,
+    };
+
+    const { error } = maintenanceWindowValidation.validate(data, {
+      abortEarly: false,
+    });
+    logger.log("error: ", error);
+    if (error) {
+      const newErrors = {};
+      error.details.forEach((err) => {
+        newErrors[err.path[0]] = err.message;
+      });
+      setErrors(newErrors);
```

Comment:
Each `Field` should be updated with errors so that it's error state is shown to the user:

![image](https://github.com/user-attachments/assets/da829e80-6b9b-4b23-87c3-5e795b6ce50e)

The error message should also be a user friendly error message, ie instead of `friendlyName cannot be empty`  it should be something like

"Friendly Name is required"

Please see other components and validation for examples.

[Login Page](https://github.com/bluewave-labs/bluewave-uptime/blob/develop/Client/src/Pages/Auth/Login.jsx) and it's associated validation is a good straightforward example.

Reply:
Done ✅

![Screenshot (115)](https://github.com/user-attachments/assets/9e6af167-7578-4424-94b6-c483d55292b5)

Reply:
![Screenshot 2024-08-21 224843](https://github.com/user-attachments/assets/c593cb11-7caa-425a-a43c-0e89698414cc)

Reply:
Maybe for this part say something like A monitor is required. Add monitors is required sounds kind of robotic lol.